src/statistics.jl

```diff
@@ -0,0 +1,46 @@
+abstract type AbstractStatistic end
+
+struct Mean{D<:AbstractDomain} <: AbstractStatistic
+   domain::D
+   values::Dict{Symbol, Vector} 
+end
+
+Mean(domain, values) = Mean{typeof(domain)}(domain, values)
+
+function Mean(solution::SimulationSolution)
+    # solution variables
+    variables = collect(keys(solution.realizations))
+
+    # output dictionary
+    values = Dict(var => mean(solution.realizations[var]) for var in variables)
```

Comment:
You could eliminate `variables` and loop the `key,val` pairs of the `realizations` dictionary:

```julia
values = Dict(var => mean(real) for (var,real) in solution.realizations)
```